compiler/stable_mir/src/crate_def.rs

```diff
@@ -50,6 +50,12 @@ pub trait CrateDef {
         let def_id = self.def_id();
         with(|cx| cx.span_of_an_item(def_id))
     }
+
+    /// Return attributes with the given attribute name.
```

Comment:
Can you please document the path format? It can just be an example on how each element of the slice is a path segment.

Reply:
Good idea to add more doc on this. It did take me a while to find out the path format myself.